generate/schemagen/gen/generator.go

```diff
@@ -78,23 +88,34 @@ func Generate(pkgRegistry PackageRegistry, linker Linker, client APIServer, opti
 		facades = append(facades, adminFacades...)
 	}
 
-	latest := make(map[string]facade.Details)
-	for _, facade := range facades {
-		if f, ok := latest[facade.Name]; ok && facade.Version < f.Version {
-			continue
+	// Compose all the facade groups together.
+	var groupFacades [][]facade.Details
+	for _, group := range opts.facadeGroups {
+		groupFacades = append(groupFacades, Filter(group, facades, registry))
+	}
+
+	unique := make(map[string]facade.Details)
+	for _, list := range groupFacades {
+		for _, f := range list {
+			unique[fmt.Sprintf("%s:%d", f.Name, f.Version)] = f
```

Comment:
is it possible to get different Details with the same name and version?

Reply:
Yeah, to ensure we only have one of each, to prevent repetition in the schema.